RetrospectiveExtension.Frontend/components/feedbackItem.tsx

```diff
@@ -689,7 +728,8 @@ class FeedbackItem extends React.Component<IFeedbackItemProps, IFeedbackItemStat
                   <button
                     title="UnVote"
                     aria-live="polite"
-                    aria-label={'Click to unvote On feedback. Current vote count is ' + this.props.upvotes}
+                    aria-label={'Click to unvote on feedback with title ' + this.props.title
```

Comment:
minor nit, I think you may be able to do the following, but it's a matter of preference. 
```
`Click to unvote on feedback with title ${this.props.title}. `
+ `Current vote count is ${this.props.upvotes}`
```